catalog-info.yaml

```diff
@@ -440,6 +440,53 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
```

Comment:
I normally suggest to create this catalog-info change in a first PR. Why? Buildkite integration with Terrazzo cannot be tested unless is merged.

So you could create a PR afterwards and point to your branch for testing purposes using the BK UI and validate changes work as expected during your PR development.